### PR TITLE
[USDU-331] Samples json update

### PR DESCRIPTION
**Pull request by lee-aandrew**
## Purpose of this PR

Quick update to the Samples.Json files and the Copyright text on Samples

**Ticket/Jira #:**

## Testing

**Functional Testing status:**
n/a

**Performance Testing status:**
n/a

## Overall Product Risks

**Complexity:**

**Halo Effect:**

## Additional information

**Note to reviewers:**

**Reminder:**

- [ ] Add entry in CHANGELOG.md _(if applicable)_

### CODE REVIEW

File: package/com.unity.formats.usd/Samples/ExportMesh/Editor/ExportMeshExampleEditor.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2017 Google Inc. All rights reserved.
+// Copyright 2023 Unity Technologies. All rights reserved.
```
Comment (vickycl) [inline package/com.unity.formats.usd/Samples/ExportMesh/Editor/ExportMeshExampleEditor.cs]: I'm not entirely sure, but I don't think we should change the copyright notice in any files we haven't changed. See: https://opensource.google/documentation/reference/copyright
The copyright is kind of a 'timestamp' of work, so as we haven't changed anything I think it should be left alone

File: package/com.unity.formats.usd/Samples/ImportMesh/Editor/UsdImportMeshExampleEditor.cs [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 Jeremy Cowles. All rights reserved.
+// Copyright 2023 Unity Technologies. All rights reserved.
```
Comment (vickycl) [inline package/com.unity.formats.usd/Samples/ImportMesh/Editor/UsdImportMeshExampleEditor.cs:1]: I think this change is probably ok

File: package/com.unity.formats.usd/Samples/ImportProcessor/CombineMeshes.cs [hunk 3]
```diff
@@ -1,3 +1,5 @@
+// Copyright 2023 Unity Technologies. All rights reserved.
```
Comment (vickycl) [inline package/com.unity.formats.usd/Samples/ImportProcessor/CombineMeshes.cs:1]: Adding it when there wasn't one before seems reasonable

File: package/com.unity.formats.usd/Samples/ExportMesh/Editor/ExportMeshExampleEditor.cs (hunk 1 above)
Comment (vickycl) [reply package/com.unity.formats.usd/Samples/ExportMesh/Editor/ExportMeshExampleEditor.cs]: According to the [Linux Foundation](https://www.linuxfoundation.org/blog/blog/copyright-notices-in-open-source-software-projects):

> Don’t change someone else’s copyright notice without their permission
You should not change or remove someone else’s copyright notice unless they have expressly (in writing) permitted you to do so. This includes third parties’ notices in pre-existing code.

I don't know whether this only applies to full Open Source though?? 😖

File: package/com.unity.formats.usd/Samples/ExportMesh/.sample.json [hunk 4]
```diff
@@ -1,4 +1,4 @@
 {
     "displayName" : "ExportMesh",
-    "description" : "A sample showing how to record a scene to a USD file."
+    "description" : "A sample showing how Unity Game Object can be exported as a USD file (with animation) using Unity's USD APIs directly."
```
Comment (vickycl) [inline package/com.unity.formats.usd/Samples/ExportMesh/.sample.json]: I think 'GameObjects' (without Unity) is enough :)

File: package/com.unity.formats.usd/Samples/UsdTimelinePlayable/.sample.json [hunk 5]
```diff
@@ -1,4 +1,4 @@
 {
     "displayName" : "UsdTimelinePlayable",
-    "description" : "A sample showing how to use the USD integration in Timeline."
+    "description" : "A sample showing how to use the USD integration within Unity's Timeline."
```
Comment (vickycl) [inline package/com.unity.formats.usd/Samples/UsdTimelinePlayable/.sample.json]: I think this should be '... USD integration _with_ Unity's Timeline'. :)

File: package/com.unity.formats.usd/Samples/ExportMesh/Editor/ExportMeshExampleEditor.cs (hunk 1 above)
Comment (lee-aandrew) [reply package/com.unity.formats.usd/Samples/ExportMesh/Editor/ExportMeshExampleEditor.cs]: hmm yeah maybe we should be on the safe side and leave this alone then - i was thinking maybe Jeremy or Julien just forgot to change this but theres the chance they just didnt change it at all

**Review by vickycl — APPROVED** [review]
Great, thanks for the updates!

**Comment by lee-aandrew** [comment]
Thanks for the review!

**Pull request merged**